elastic/integrations
# Nginx: refactor dashboards

## What does this PR do?



This PR refactors the Nginx dashboards according to the migration assessment. Changes:
- [x] Add the total requests visualization
- [x] Add the processed connections visualization
- [x] Add the uptime visualization
- [x] Add the Nginx instance dropdown selector (filter)
- [x] Use margins between the panels
- [x] Remove points from each line (i think it's done via point size input)

## Checklist


## How to test this PR locally



Run the Elastic stack and the Nginx docker container according to the howto: https://github.com/elastic/elastic-package/blob/master/docs/howto/connect_service_with_elastic_stack.md
Verify if data are presented on dashboards. 

## Related issues


- https://github.com/elastic/integrations/issues/373

## Screenshots



![screencapture-localhost-5601-app-dashboards-2020-12-10-16_36_29](https://user-images.githubusercontent.com/14044910/101794329-0d5de800-3b07-11eb-9d1c-cd5a65f99bdc.png)
![screencapture-localhost-5601-app-dashboards-2020-12-10-16_37_01](https://user-images.githubusercontent.com/14044910/101794344-10f16f00-3b07-11eb-86ad-4e1f2fe45237.png)
![screencapture-localhost-5601-app-dashboards-2020-12-10-16_37_32](https://user-images.githubusercontent.com/14044910/101794358-13ec5f80-3b07-11eb-9bb0-67fd00d12b4c.png)
